addons/web/static/src/views/list/list_renderer.js

```diff
@@ -61,6 +61,7 @@ import { exprToBoolean } from "@web/core/utils/strings";
 const formatters = registry.category("formatters");
 
 const DEFAULT_GROUP_PAGER_COLSPAN = 1;
+const MOVABLE_RECORD_TYPES = ["char", "boolean", "integer", "selection", "many2one"];
```

Comment:
maybe move this constant to relational_model.js ? It's the same as for kanban...